### ISSUE TITLE
Time to mark virtualmin-svn as deprecated? 

### ISSUE THREAD

**Issue by marclaporte** [virtualmin/virtualmin-svn#2]
Related: https://github.blog/2023-01-20-sunsetting-subversion-support/

**Comment by marclaporte** [comment]
Related: https://forum.virtualmin.com/t/subversion-plugin-missing-from-repository-on-ubuntu-20-04-or-higher/128314

**Comment by jcameron** [comment]
Yeah this can probably be at least removed from the default install script

**Comment by SubJunk** [comment]
FWIW I am both a Git user and a SVN user, because SVN has better support for large files. There is an extension to Git called Git LFS but without using extensions it's quite limited.
